### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by son-vyas**

### CODE REVIEW

**Comment by kotp** [comment]
ref #754

File: README.md [hunk 1]
```diff
@@ -109,6 +109,28 @@ tree -L 1 ~/code/exercism
 ├── problem-specifications
 └── ruby
 ```
+To explain a bit more, you must follow this commands step-by-step:-
```
Comment (kotp) [inline README.md]: `follow these commands`

File: README.md [hunk 2]
```diff
@@ -109,6 +109,28 @@ tree -L 1 ~/code/exercism
...
+
+``` bash
+  $ mkdir exercism
+  $ cd exercism
+```
+Fork the [exercism/ruby](https://github.com/exercism/ruby)
+
+Fork the [exercism/problem-specifications](https://github.com/exercism/problem-specifications)
+
+Now you need to clone both the above repositories
```
Comment (kotp) [inline README.md]: `Now clone these two repositories` maybe, but for sure the `you need to` can go.

File: README.md [hunk 3]
```diff
@@ -109,6 +109,28 @@ tree -L 1 ~/code/exercism
...
+``` bash
+
+$ git clone https://github.com/YOUR-USERNAME/YOUR-RUBY-REPOSITORY
+$ git clone https://github.com/YOUR-USERNAME/YOUR-PROBLEM-S-REPOSITORY
+
+```
+Next, you need to [configure the remote](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and [synchronize](https://help.github.com/articles/syncing-a-fork/) it.
```
Comment (kotp) [inline README.md]: `you need to` can again go.

File: README.md [hunk 4]
```diff
@@ -109,6 +109,28 @@ tree -L 1 ~/code/exercism
...
+
+Make sure you have synced up local master branch and upstream master branch.
```
Comment (kotp) [inline README.md]: Why and when does this step need to be done?

**Comment by son-vyas** [comment]
yeah @kotp I will do explain why the sync is required.

**Comment by son-vyas** [comment]
@kotp , done with the sync cause

**Comment by Insti** [comment]
Thanks @son-vyas.

**Pull request merged**